clang/test/dpct/cmake_migation/case_023/case_023_cuda_compile_ptx.cpp

```diff
@@ -0,0 +1,10 @@
+// RUN: rm -rf %T && mkdir -p %T
+// RUN: cd %T
+// RUN: cp %S/input.cmake ./input.cmake
+// RUN: dpct -in-root ./ -out-root out  ./input.cmake --rule-file=%S/../../../../tools/dpct/DpctOptRules/cmake_script_migration_rule.yaml --migrate-cmake-script-only --cuda-include-path="%cuda-path/include"
```

Comment:
Please follow up to refine: 
1. if --migrate-cmake-script-only used, then  --cuda-include-path="%cuda-path/include" is not needed. 
2. cmake_script_migration_rule.yaml should be loaded auto

Reply:
Will submit a separate PR to refer them.